CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased - as of Sprint 53 end 2017-01-30
```

Comment:
Based on this output from above.  I have removed #61 and added #66. Also changed title inside file to just "unreleased"